features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/tiers/tiers.jag

```diff
@@ -22,6 +22,29 @@ var getTiers = function () {
     }
 };
 
+var getAppTiers = function () {
+    var tiers,
+            log = new Log(),
```

Comment:
fixed in commit: https://github.com/ruks/carbon-apimgt/commit/7700df9b0b9643da2e59195cbbc6d681b8514caf